### PR TITLE
Remove erl_interface and try to use new pg module

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

**Comment by Vagabond** [comment]
This passes for me if I use `dbus-launch ./rebar3 ct` on otp 23

File: c_src/Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ endif
 CFLAGS += -fPIC -I $(ERTS_INCLUDE_DIR) -I $(ERL_INTERFACE_INCLUDE_DIR)
 CXXFLAGS += -fPIC -I $(ERTS_INCLUDE_DIR) -I $(ERL_INTERFACE_INCLUDE_DIR)
 
-LDLIBS += -L $(ERL_INTERFACE_LIB_DIR) -lerl_interface -lei -ldbus-1
```
Comment (evanmcc) [inline c_src/Makefile:42]: @JayKickliter has asked that we remove ei too, since NIFs don't use it and that might save us some trouble down the road.

Comment (Vagabond) [reply c_src/Makefile:42]: No objections.

Comment (evanmcc) [reply c_src/Makefile:42]: done

**Review by evanmcc — APPROVED** [review]
added 23 to travis for this branch, tests are passing with each version.  removed ei, will merge assuming this test run also passes cleanly.

**Review by madninja — APPROVED** [review]

**Pull request merged**